Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
> We are compiling it for 64 bit

My understanding is that the MonoMac compiler/VM ignores that setting...  it only generates 32bit code.